public_suffix_list.dat

```diff
@@ -867,6 +867,31 @@ int.cv
 nome.cv
 org.cv
 
+// CV DOIT PERSADA INDONESIA : https://doit.co.id
```

Comment:
Andhy please move the whole submitted section out of the ICANN section and down into the #PRIVATE section ordered by where your organization's name would be the next alphabetical org based upon other entries' commented organization name.

Reply:
I just edited too, thank you